common/static/common/js/components/views/paging_mixin.js

```diff
@@ -1,37 +1,40 @@
-define([],
-    function () {
-        var PagedMixin = {
-            setPage: function (page) {
-                var self = this,
-                    collection = self.collection,
-                    oldPage = collection.currentPage;
-                collection.goTo(page, {
-                    reset: true,
-                    success: function () {
-                        window.scrollTo(0, 0);
-                    },
-                    error: function (collection) {
-                        collection.currentPage = oldPage;
-                        self.onError();
+;(function (define) {
```

Comment:
We're only using PagingMixin in one place in the code right now (`cms/static/js/views/paged_container.js`). Do we really need to keep it as a mixin, or can we just plop this code into `PagedContainerView`?